src/constants/graphqlQueries.ts

```diff
@@ -26,12 +26,18 @@ export const generateL2RasterProductQuery = `
         }
     }
 `
+// , $cycle: Int, $pass: Int, $scene: Int, $outputGranuleExtentFlag: Boolean, $outputSamplingGridType: GridType, $beforeTimestamp: String, $afterTimestamp: String
+// , cycle: $cycle, pass: $pass, scene: $scene, outputGranuleExtentFlag: $outputGranuleExtentFlag, outputSamplingGridType: $outputSamplingGridType, beforeTimestamp: $beforeTimestamp, afterTimestamp: $afterTimestamp
 
 export const userProductsQuery = `
-    query getUserProducts($limit: Int, $after: ID) 
+    query getUserProducts($limit: Int, $after: ID, $cycle: Int, $pass: Int, $scene: Int, $outputGranuleExtentFlag: Boolean, $outputSamplingGridType: GridType, $beforeTimestamp: String, $afterTimestamp: String) 
     {
         currentUser {
+<<<<<<< HEAD
+            products (limit: $limit, after: $after, cycle: $cycle, pass: $pass, scene: $scene, outputGranuleExtentFlag: $outputGranuleExtentFlag, outputSamplingGridType: $outputSamplingGridType, beforeTimestamp: $beforeTimestamp, afterTimestamp: $afterTimestamp) {
+=======
             products (limit: $limit, after: $after) {
+>>>>>>> e84f20048ee552f578368e0d47bcc3bac894afd2
```

Comment:
Unfinished git merge?

Reply:
Yes, I must have missed this. The merge editor was not showing this one